scripts/popup.js

```diff
@@ -23,7 +23,42 @@ function view_all_function(){
 	document.location.href = open_url;
 }
 
+function facebook_function(){
+	var link = "https://www.facebook.com/sharer/sharer.php?u=";
+	chrome.tabs.getSelected(null,function(tab) {
+		chrome.windows.create({'url': link+tab.url, 'type': 'popup', 'width' : 500, 'height' : 400}, function(window) {
+		});
+	});
+}
+
+function google_function(){
+	var link = "https://plus.google.com/share?url=";
+	chrome.tabs.getSelected(null,function(tab) {
+		chrome.windows.create({'url': link+tab.url, 'type': 'popup', 'width' : 500, 'height' : 400}, function(window) {
+		});
```

Comment:
@rgpt thanks for the reply again. But buddy you are still not clear with your words. 
In your previous comment, you are suggesting about sharing the search on Wayback's page while in this comment you are talking about search results.
If you are suggesting about adding the social support with search integration, then, that is possible after fusing the two features together.
Secondly, for now, on clicking any of the social share buttons, it just gets the current URL ( which supposedly will be Wayback Machine's archived URL ) and shares it on the specified social websites. 

If this still doesn't satisfy your doubt, feel free to contact me on [fb](https://www.facebook.com/shubham.mangal00).
And lastly, I really appreciate your concern and hard work you are doing on reviewing the code. This is really leading to helpful and fruitful discussions.